src/core/connection.c

```diff
@@ -6820,6 +6820,31 @@ QuicConnParamGet(
     uint8_t Type;
 
     switch (Param) {
+    
+    case QUIC_PARAM_CONN_ORIG_DEST_CID:
+        if (Connection == NULL || Connection->OrigDestCID == NULL) {
+            Status = QUIC_STATUS_INVALID_STATE;
+            break;
+        }
+        if (Buffer == NULL) {
+            Status = QUIC_STATUS_INVALID_PARAMETER;
+            break;
+        }
+        if (*BufferLength < Connection->OrigDestCID->Length) {
+            // Tell app it needs to pass in a bigger buffer.
```

Comment:
We don't need a comment here.

BTW, the comment style in msquic is:
```
//
// This is a comment.
//
```

Reply:
And please apply this comment style to all the comments you added in the PR.

Reply:
can do